src/main/scala/com/ibm/aardpfark/pfa/expression/FunctionCalls.scala

```diff
@@ -1,7 +1,7 @@
 package com.ibm.aardpfark.pfa.expression
 
 import com.ibm.aardpfark.pfa.document.{PFAExpressionSerializer, ParamSerializer, SchemaSerializer}
-import org.json4s.native.JsonMethods.parse
+import org.json4s.native.JsonMethods.{parse => parseJSON}
```

Comment:
the dsl._ namespace has `parse.{int,float..}` functions that conflict with JsonMethods